R/tm_a_regression.R

```diff
@@ -34,13 +34,17 @@
 #' # Regression graphs from selected response variable (BMRKR1) and
 #' # selected regressors (AGE)
 #'
-#' ADSL <- teal.modules.general::rADSL
+#' data <- teal_data()
+#' data <- within(data, {
+#'   library(nestcolor)
+#'   ADSL <- teal.modules.general::rADSL
+#' })
+#' datanames <- c("ADSL")
+#' datanames(data) <- datanames
+#' data@join_keys <- cdisc_join_keys(!!!datanames)
```

Comment:
I see the point now of waiting for change of the join_keys. Let's continue, maybe we will be able to merge them in the same time.